src/util.ts

```diff
@@ -33,7 +35,10 @@ const maxDepthDefault = 5;
  * @param object The object to materialize
  * @param depth The maximum depth of materialization (maxDepthDefault = 5)
  */
-export async function materialize(object?: firestore.DocumentData, depth = maxDepthDefault): Promise<any> {
+export async function materialize(
```

Comment:
what does materialize do?

Reply:
I believe `materialize` will take a JSON that has references to other firebase documents and replaces the reference with the actual data from that document